src/options.ts

```diff
@@ -130,8 +130,7 @@ export async function processOptions(
       if (s.isFile()) {
         const pathParts = options.path[0].split(path.sep);
         options.path = [path.sep + pathParts[pathParts.length - 1]];
-        options.serverRoot =
-          pathParts.slice(0, pathParts.length - 1).join(path.sep) || '.';
+        options.serverRoot = pathParts.slice(0, -1).join(path.sep) || '.';
```

Comment:
This does not seem correct.

Reply:
Are you sure? From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) I see that end can be a negative number:

>A negative index can be used, indicating an offset from the end of the sequence.

Reply:
OH!  I misunderstood what was happening because the diff was hard to read - apologies!  This is good 👍